manifests/v2/base/runtimes/pre-training/torch-distributed.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kubeflow.org/v2alpha1
+kind: ClusterTrainingRuntime
+metadata:
+  name: torch-distributed
```

Comment:
It would be useful to anticipate support for other accelerators like AMD/ROCm, so calling that one like `torch-cuda-distributed` could help to be more descriptive.

Reply:
I think, it is a good idea.
However, we are still discussing with @tenzen-y on how we should propagate accelerator type to the runtime labels given that it overlaps with Kueue ResourceFlavor concepts.
As you can see, by default we don't attach any resources to the Trainer container in this pre-training runtime: https://github.com/kubeflow/training-operator/blob/bc7af4e2aee6dec0fdf01c9ad145fc997ec30a10/manifests/v2/base/runtimes/pre-training/torch-distributed.yaml#L21

@astefanutti Could you create a separate issue so we can track it ?
E.g. support AMD accelerators in Torch Distributed Training Runtime.

Reply:
@andreyvelich thanks for the feedback, I've just created #2335.

In addition to the resources, it's also the container image set for the runtime that's coupled to the accelerator.

For instance `pytorch/pytorch:2.5.0-cuda12.4-cudnn9-runtime` makes that training runtime specific to NVIDIA CUDA.

For AMD ROCm, that would need to be something like `rocm/pytorch:rocm6.2.3_ubuntu22.04_py3.10_pytorch_release_2.3.0` for example.

Reply:
Yeah, that is correct. We need to find image that adds support for ROCm.